src/main.zig

```diff
@@ -1,37 +1,40 @@
 const std = @import("std");
 const builtin = @import("builtin");
 const gpu = @import("mach-gpu");
-const vulkan = @import("vulkan.zig");
 
 var inited = false;
 var allocator: std.mem.Allocator = undefined;
+const impl = switch (builtin.os.tag) {
+    .linux => @import("vulkan.zig"),
+    .windows, .macos, .ios => @compileError("TODO: unsupported platform"),
```

Comment:
All platforms should return vulkan for the time being, or at the very least Windows should return Vulkan, im not sure whether we should support MoltenVK until we have dedicated a Metal backend